lib/packwerk/parsers/erb.rb

```diff
@@ -44,7 +44,7 @@ def parse_buffer(buffer, file_path:)
         params(
           erb_ast: T.all(::AST::Node, Object),
           file_path: String
-        ).returns(::AST::Node)
+        ).returns(T.nilable(::AST::Node))
```

Comment:
Is it possible to add coverage? If it isn't covered in our test suite I can see this being removed again.

Reply:
Yep, I can look more into it.

Reply:
It looks like ERB can contain javascript which returns `nil` rather than a node. I added a simple test to demonstrate this. I also added more types to the parser class so that the nil type is also supported by the return value of the parser classes.